### ISSUE TITLE
examples/clickstream

### ISSUE THREAD

**Issue by haf** [confluentinc/examples#833]
Is this quote char intended at the end?

https://github.com/confluentinc/examples/blob/6.0.0-post/clickstream/docker-compose.yml#L81

**Comment by ybyzek** [comment]
@haf good catch!  This is not intended, thank you for surfacing this.  I will happily fix unless you're interested in submitting a PR?
